Files/UserControls/LayoutModes/GridViewBrowser.xaml

```diff
@@ -419,7 +422,7 @@
                 HorizontalAlignment="Stretch"
                 VerticalAlignment="Stretch"
                 Background="Transparent"
-                DataContext="{x:Bind}"
+                DataContext="{Binding}"
```

Comment:
Is it necessary to use `Binding` instead of `x:Bind` here?

Reply:
If I use DataContext={x:Bind} plus the added WrapPanel the app crashes with StackOverflowException. Why? I have no clue